src/xai_grok/schemas/message_image_source.py

```diff
@@ -0,0 +1,7 @@
+import pydantic
+
+
+class MessageImageSource(pydantic.BaseModel):
+    data: str
+    media_type: str
+    type: str
```

Comment:
Using `type` as a field name shadows the built-in `type` function, which can lead to unexpected behavior. Consider renaming the field to avoid conflicts.